src/cryptography/hazmat/backends/commoncrypto/ciphers.py

```diff
@@ -151,6 +160,8 @@ def __init__(self, backend, cipher, mode, operation):
         self.authenticate_additional_data(b"")
 
     def update(self, data):
+        self._plaintext_size_validator.update(data)
+        self._plaintext_size_validator.validate()
```

Comment:
If we always call `validate` right after `update` why do we need that part of the API at all?